src/test/kotlin/blackjack/domain/game/BlackJackTest.kt

```diff
@@ -0,0 +1,98 @@
+package blackjack.domain.game
+
+import blackjack.domain.card.TestCards
+import blackjack.domain.participant.Dealer
+import blackjack.domain.participant.Player
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+internal class BlackJackTest {
+    @Test
+    internal fun `게임이 시작되면 플레이어에게 두장의 카드가 주어진다`() {
+        val player = Player("pobi")
+        player.cards.values.size shouldBe 0
+        val blackJack = BlackJack(listOf(player))
+        blackJack.distributeInitialCard()
+        player.cards.values.size shouldBe 2
+        blackJack.dealer.cards.values.size shouldBe 2
```

Comment:
개인적으로 테스트 코드를 작성할 때는 given, when, then 구조를 지키는 편인데, 
테스트 코드를 파악하기 쉽도록 아래와 같이 주석(= 주석은 개인취향!) 혹은 줄바꿈을 통해서 given, when, then을 한 눈에 파악할 수 있도록 구성해주시면 좋을 것 같아요. 😃

```suggestion
        // given
        val player = Player("pobi")
        player.cards.values.size shouldBe 0
        val blackJack = BlackJack(listOf(player))
        
        // when
        blackJack.distributeInitialCard()
        
        // then
        player.cards.values.size shouldBe 2
        blackJack.dealer.cards.values.size shouldBe 2
```